utilities/validations.py

```diff
@@ -24,6 +24,8 @@ def validate_empty_fields(**kwargs):
         value = kwargs.get(field)
         if not type(value) is bool and not value:
             raise AttributeError(field + " is required field")
+        if isinstance(value, str) and not value.strip():  # remove white space
```

Comment:
Hello @verenceLola, great work here. Kindly remove the spaces at the ends too if that is possible.

Reply:
@nakatuddesuzan I'm not sure what you mean here. Kindly shade more colour on this. Thanks

Reply:
<img width="1093" alt="Screen Shot 2019-03-11 at 21 30 49" src="https://user-images.githubusercontent.com/39129473/54148365-2246ac80-4445-11e9-881d-eedbc4efb285.png">

<img width="1153" alt="Screen Shot 2019-03-11 at 21 31 12" src="https://user-images.githubusercontent.com/39129473/54148367-2246ac80-4445-11e9-966b-25a3c4bf8dba.png">
checkout those screenshots. The same question is created, the difference is that the second has spaces at the end. Try it out and see what am talking about.